src/cli.ts

```diff
@@ -57,6 +57,7 @@ async function main(argv: string[]) {
     .option("--default-tape <tape-name>", "Name of the default tape", "default")
     .option("-h, --host <host>", "Host to proxy (not required in replay mode)")
     .option("-p, --port <port>", "Local port to serve on", "3000")
+    .option("--match <match>", "Strict request matching")
```

Comment:
**Suggestion**: The CLI argument here could be a lot more self-descriptive:

```suggestion
    .option("--exact-request-matching", "Perform exact request matching instead of best-effort request matching during replay.")
```

Reply:
Done